input/pagecontent/3-15-diagnosticreport-select.md

```diff
@@ -0,0 +1,58 @@
+eventMaturity | [1 - Submitted](../../specification/STU3/#event-maturity-model)
+
+### Workflow
+A `DiagnosticReport-select` request will be made to the Hub when an application desires to indicate that one or more FHIR resources contained in the anchor context's content are to be made visible, in focus, or otherwise "selected". It is assumed that a FHIR resource (e.g., observation) with the specified `id` is contained in the current anchor context's content, the Hub MAY or MAY NOT provide validation of its presence.
```

Comment:
Why is the -select action limited to only content-sharing scenarios?

Reply:
I am still not sure how to raise -select out of the current anchor context.  With multiple contexts if there is a context.id then a select could be a "Context-select" with the context.id provided and an optional select array?

How else could it work?

Without a context.id would we just say that whatever the context resource is:

    "context": [
      {
        "key": "report",
        "resource": {
          "resourceType": "DiagnosticReport",
          "id": "40012366"
        }
      },

then the context is uniquely identified by the resource's id and the operation could still be "Context-select"?